app/components/application/details/application-details.html

```diff
@@ -0,0 +1,59 @@
+<bex-loading [status]="data.newDataStatus | async">
+    <div *ngIf="application">
+        <md-card class="overview">
+            <md-card-title-group style="margin: 0px;">
+                <div style="align-items:center;">
+                    <bex-refresh-btn [refresh]="refresh"></bex-refresh-btn>
+                </div>
+                <div style="margin-left:12px; margin-right:auto;">
+                    <md-card-title>{{decorator.id}}</md-card-title>
+                    <md-card-subtitle>
+                        <i class="fa" [ngClass]="application.allowUpdates ? 'fa-unlock-alt' : 'fa-lock'"></i> {{decorator.displayName}}
+                        <div *ngIf="application.allowUpdates"> <b>(unlocked)</b></div>
+                        <div *ngIf="!application.allowUpdates"> <b>(locked)</b></div>
+                    </md-card-subtitle>
+                </div>
+            </md-card-title-group>
+            <md-card-content></md-card-content>
+            <md-card-actions layout="row" layout-align="end center">
+                <bex-add-button title="Add package" (click)="addPackage()"></bex-add-button>
+                <bex-edit-button (click)="editApplication()"></bex-edit-button>
+                <bex-delete-button [entity]="application" (click)="deleteApplication()"></bex-delete-button>
+            </md-card-actions>
+        </md-card>
+        <bex-application-error-display [application]="application"></bex-application-error-display>
+        <md-card class="details">
+            <bex-tab-group>
+                <bex-tab key="properties">
+                    <bex-tab-label>Properties</bex-tab-label>
+                    <bex-application-properties [application]="application"></bex-application-properties>
+                </bex-tab>
+                <bex-tab key="packages">
+                    <bex-tab-label>Packages</bex-tab-label>
+                    <bex-entity-details-list #listWrapper [showRefreshButton]="false" [showExpandButton]="false" [showFilterIcon]="false"
```

Comment:
its only really nasty because of the extra commands. 
i could bundle the commands up into their own component?